pupil-spa/src/app/services/feedback/feedback.service.ts

```diff
@@ -19,7 +19,7 @@ export class FeedbackService {
       const accessToken = this.storageService.getItem('access_token');
 
       if (!storedFeedback || !accessToken) {
-        return reject('Missing data in local storage');
+        return false;
```

Comment:
I just noticed that this is wrapper in both a promise and an async func.
I think one of these two is only required.
I'd keep just the async/await for simplicity, see the example on submission.service.ts